specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2019-01-21/kusto.json

```diff
@@ -299,6 +299,105 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Kusto/clusters/{clusterName}/listFollowerDatabases": {
+      "post": {
+        "tags": [
+          "Clusters"
+        ],
+        "description": "Returns a list of databases that are owned by this cluster and were followed by another cluster.",
+        "operationId": "Clusters_ListFollowerDatabases",
+        "x-ms-examples": {
+          "KustoClusterListFollowerDatabases": {
+            "$ref": "./examples/KustoClusterListFollowerDatabases.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of followed databases.",
+            "schema": {
+              "$ref": "#/definitions/FollowerDatabaseListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Kusto/clusters/{clusterName}/detachFollowerDatabases": {
+      "post": {
+        "tags": [
+          "Clusters"
+        ],
+        "description": "Detaches all followers of a database owned by this cluster.",
+        "operationId": "Clusters_DetachFollowerDatabases",
+        "x-ms-examples": {
+          "KustoClusterDetachFollowerDatabases": {
+            "$ref": "./examples/KustoClusterDetachFollowerDatabases.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "followerDatabasesToRemove",
```

Comment:
The property name here is plural, but the request object only takes a single clusterId + databaseName tuple. Should this take an array of tuples?

Reply:
You're right I'll rename the parameter to singular (I rather keep the API simple)